api/Document.json

```diff
@@ -5359,6 +5359,76 @@
           }
         }
       },
+      "prerendering": {
+        "__compat": {
+          "spec_url": "https://wicg.github.io/nav-speculation/prerendering.html#dom-document-prerendering",
+          "support": {
+            "chrome": {
+              "version_added": "108"
+            },
+            "chrome_android": "mirror",
+            "edge": "mirror",
+            "firefox": {
+              "version_added": false
+            },
+            "firefox_android": "mirror",
+            "ie": {
+              "version_added": false
+            },
+            "oculus": "mirror",
+            "opera": "mirror",
+            "opera_android": "mirror",
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": "mirror",
+            "samsunginternet_android": "mirror",
+            "webview_android": "mirror"
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "prerenderingchange_event": {
+        "__compat": {
+          "description": "<code>prerenderingchange</code> event",
```

Comment:
I get an error running the test: https://mdn-bcd-collector.gooborg.com/tests/api/Document/prerenderingchange_event "Could not find a test for api.Document.prerenderingchange_event." on chrome